README.md

```diff
@@ -2,52 +2,7 @@
 [![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=uzhase_bnbexplorer-backend&metric=alert_status)](https://sonarcloud.io/dashboard?id=uzhase_bnbexplorer-backend)
 [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=uzhase_bnbexplorer-backend&metric=coverage)](https://sonarcloud.io/dashboard?id=uzhase_bnbexplorer-backend)
 
-# Swagger generated server
```

Comment:
I think we should keep this content too, its always useful to have run instructions (if someone only wants to run the backend) in the readme.

Reply:
@cdeiac Done